tests/unit/development/test_base_component.py

```diff
@@ -473,3 +474,37 @@ def test_debc027_component_error_message():
         + "keyword argument: `asdf`\n"
         + "Allowed arguments: {}".format(", ".join(sorted(html.Div()._prop_names)))
     )
+
+
+def test_set_random_id():
+    app = Dash(__name__)
+
+    input1 = dcc.Input(value="Hello")
+    input2 = dcc.Input(value="Hello")
```

Comment:
```suggestion
    input1 = dcc.Input(value="Hello Input 1")
    input2 = dcc.Input(value="Hello Input 2")
```

Just to keep things unambiguous